packages/oas-utils/src/entities/spec/request-examples.ts

```diff
@@ -149,6 +150,7 @@ export const exampleRequestBodySchema = z.object({
     .object({
       encoding: z.enum(exampleRequestBodyEncoding),
       value: z.string().default(''),
+      mimeType: z.string().optional(),
```

Comment:
we should probably make a schema off `contenttype` eh @amritk ?

Reply:
what kind of schema you thinking? because there's way too many to add an enum